src/sentry/static/sentry/app/views/settings/components/forms/formField/index.tsx

```diff
@@ -14,13 +14,15 @@ import FormFieldControlState from 'app/views/settings/components/forms/formField
 import PanelAlert from 'app/components/panels/panelAlert';
 import ReturnButton from 'app/views/settings/components/forms/returnButton';
 import space from 'app/styles/space';
+import Alert from 'app/components/alert';
+import FormModel from 'app/views/settings/components/forms/model';
 
 /**
  * Some fields don't need to implement their own onChange handlers, in
  * which case we will receive an Event, but if they do we should handle
  * the case where they return a value as the first argument.
  */
-const getValueFromEvent = (valueOrEvent, e) => {
+const getValueFromEvent = (valueOrEvent?: any, e?: any) => {
```

Comment:
Could `e` be typed as `MouseEvent`?

Reply:
@markstory It can but we are passing in `...args` every place it's used so it's it doesn't really add any benefit. In a future pass, we might want to tighten this up.